geometry_msgs/Pose2D pose
float32 l
float32 w
float32 h
